INFO 2022-08-31 14:16:54,833 autoreload 677 Watching for file changes with StatReloader
INFO 2022-08-31 14:21:48,558 autoreload 266 D:\Users\18203\Desktop\work\blogproject\blog\models.py changed, reloading.
INFO 2022-08-31 14:21:49,517 autoreload 677 Watching for file changes with StatReloader
INFO 2022-08-31 14:22:31,621 autoreload 266 D:\Users\18203\Desktop\work\blogproject\blog\models.py changed, reloading.
INFO 2022-08-31 14:22:32,417 autoreload 677 Watching for file changes with StatReloader
INFO 2022-08-31 14:22:55,305 autoreload 266 D:\Users\18203\Desktop\work\blogproject\blog\models.py changed, reloading.
INFO 2022-08-31 14:22:55,998 autoreload 677 Watching for file changes with StatReloader
INFO 2022-08-31 14:23:16,712 autoreload 266 D:\Users\18203\Desktop\work\blogproject\blog\models.py changed, reloading.
INFO 2022-08-31 14:23:17,536 autoreload 677 Watching for file changes with StatReloader
INFO 2022-08-31 14:25:18,616 autoreload 266 D:\Users\18203\Desktop\work\blogproject\blog\models.py changed, reloading.
INFO 2022-08-31 14:25:19,458 autoreload 677 Watching for file changes with StatReloader
INFO 2022-08-31 14:25:46,558 autoreload 266 D:\Users\18203\Desktop\work\blogproject\blog\models.py changed, reloading.
INFO 2022-08-31 14:25:47,447 autoreload 677 Watching for file changes with StatReloader
INFO 2022-08-31 14:27:30,286 autoreload 266 D:\Users\18203\Desktop\work\blogproject\blogproject\urls.py changed, reloading.
INFO 2022-08-31 14:27:31,018 autoreload 677 Watching for file changes with StatReloader
INFO 2022-08-31 14:27:56,021 autoreload 266 D:\Users\18203\Desktop\work\blogproject\blogproject\urls.py changed, reloading.
INFO 2022-08-31 14:27:56,776 autoreload 677 Watching for file changes with StatReloader
INFO 2022-08-31 14:29:50,243 basehttp 187 "GET / HTTP/1.1" 200 18175
INFO 2022-08-31 14:29:54,950 basehttp 187 "GET /all/rss/ HTTP/1.1" 200 276245
INFO 2022-08-31 14:30:45,965 autoreload 677 Watching for file changes with StatReloader
INFO 2022-08-31 14:30:48,567 basehttp 187 "GET / HTTP/1.1" 200 18175
ERROR 2022-08-31 14:30:52,627 log 241 Internal Server Error: /posts/5/
Traceback (most recent call last):
  File "D:\Users\18203\blog\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Users\18203\blog\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Users\18203\blog\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "D:\Users\18203\blog\lib\site-packages\django\views\generic\base.py", line 142, in dispatch
    return handler(request, *args, **kwargs)
  File "D:\Users\18203\Desktop\work\blogproject\blog\views.py", line 59, in get
    response = super().get(request, *args, **kwargs)
  File "D:\Users\18203\blog\lib\site-packages\django\views\generic\detail.py", line 108, in get
    self.object = self.get_object()
  File "D:\Users\18203\Desktop\work\blogproject\blog\views.py", line 80, in get_object
    post.toc = m.group(1) if m is not None else ''
AttributeError: can't set attribute
ERROR 2022-08-31 14:30:52,629 basehttp 187 "GET /posts/5/ HTTP/1.1" 500 88787
INFO 2022-08-31 14:31:44,764 autoreload 677 Watching for file changes with StatReloader
INFO 2022-08-31 14:31:46,635 basehttp 187 "GET / HTTP/1.1" 200 18175
ERROR 2022-08-31 14:31:49,203 log 241 Internal Server Error: /posts/5/
Traceback (most recent call last):
  File "D:\Users\18203\blog\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Users\18203\blog\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Users\18203\blog\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "D:\Users\18203\blog\lib\site-packages\django\views\generic\base.py", line 142, in dispatch
    return handler(request, *args, **kwargs)
  File "D:\Users\18203\Desktop\work\blogproject\blog\views.py", line 59, in get
    response = super().get(request, *args, **kwargs)
  File "D:\Users\18203\blog\lib\site-packages\django\views\generic\detail.py", line 108, in get
    self.object = self.get_object()
  File "D:\Users\18203\Desktop\work\blogproject\blog\views.py", line 80, in get_object
    post.toc = m.group(1) if m is not None else ''
AttributeError: can't set attribute
ERROR 2022-08-31 14:31:49,205 basehttp 187 "GET /posts/5/ HTTP/1.1" 500 88787
INFO 2022-08-31 14:36:26,592 autoreload 266 D:\Users\18203\Desktop\work\blogproject\blog\models.py changed, reloading.
INFO 2022-08-31 14:36:27,380 autoreload 677 Watching for file changes with StatReloader
ERROR 2022-08-31 14:36:36,090 log 241 Internal Server Error: /posts/5/
Traceback (most recent call last):
  File "D:\Users\18203\blog\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Users\18203\blog\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Users\18203\blog\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "D:\Users\18203\blog\lib\site-packages\django\views\generic\base.py", line 142, in dispatch
    return handler(request, *args, **kwargs)
  File "D:\Users\18203\Desktop\work\blogproject\blog\views.py", line 59, in get
    response = super().get(request, *args, **kwargs)
  File "D:\Users\18203\blog\lib\site-packages\django\views\generic\detail.py", line 108, in get
    self.object = self.get_object()
  File "D:\Users\18203\Desktop\work\blogproject\blog\views.py", line 80, in get_object
    post.toc = m.group(1) if m is not None else ''
AttributeError: can't set attribute
ERROR 2022-08-31 14:36:36,092 basehttp 187 "GET /posts/5/ HTTP/1.1" 500 88924
ERROR 2022-08-31 14:36:38,778 log 241 Internal Server Error: /posts/5/
Traceback (most recent call last):
  File "D:\Users\18203\blog\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Users\18203\blog\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Users\18203\blog\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "D:\Users\18203\blog\lib\site-packages\django\views\generic\base.py", line 142, in dispatch
    return handler(request, *args, **kwargs)
  File "D:\Users\18203\Desktop\work\blogproject\blog\views.py", line 59, in get
    response = super().get(request, *args, **kwargs)
  File "D:\Users\18203\blog\lib\site-packages\django\views\generic\detail.py", line 108, in get
    self.object = self.get_object()
  File "D:\Users\18203\Desktop\work\blogproject\blog\views.py", line 80, in get_object
    post.toc = m.group(1) if m is not None else ''
AttributeError: can't set attribute
ERROR 2022-08-31 14:36:38,779 basehttp 187 "GET /posts/5/ HTTP/1.1" 500 88787
INFO 2022-08-31 14:39:22,210 autoreload 677 Watching for file changes with StatReloader
INFO 2022-08-31 14:39:25,485 basehttp 187 "GET /posts/5/ HTTP/1.1" 200 29816
WARNING 2022-08-31 14:39:25,563 log 241 Not Found: /posts/5/example.jpg
WARNING 2022-08-31 14:39:25,563 basehttp 187 "GET /posts/5/example.jpg HTTP/1.1" 404 3369
WARNING 2022-08-31 14:39:25,565 log 241 Not Found: /posts/5/a.jpg
WARNING 2022-08-31 14:39:25,566 basehttp 187 "GET /posts/5/a.jpg HTTP/1.1" 404 3351
INFO 2022-08-31 14:39:35,380 basehttp 187 "GET /all/rss/ HTTP/1.1" 200 276245
INFO 2022-08-31 14:40:55,344 basehttp 187 "GET /all/rss/ HTTP/1.1" 200 276245
INFO 2022-08-31 14:41:58,210 basehttp 187 "GET /all/rss/ HTTP/1.1" 200 276245
INFO 2022-08-31 14:41:59,069 basehttp 187 "GET /all/rss/ HTTP/1.1" 200 276245
WARNING 2022-08-31 14:41:59,184 log 241 Not Found: /posts/5/example.jpg
WARNING 2022-08-31 14:41:59,185 basehttp 187 "GET /posts/5/example.jpg HTTP/1.1" 404 3369
INFO 2022-08-31 14:42:12,100 basehttp 187 "GET /posts/60/ HTTP/1.1" 200 14660
INFO 2022-08-31 14:42:23,428 basehttp 187 "GET /posts/151/ HTTP/1.1" 200 13761
INFO 2022-08-31 14:42:31,533 basehttp 187 "GET /posts/138/ HTTP/1.1" 200 15377
INFO 2022-08-31 14:42:45,100 basehttp 187 "GET /posts/146/ HTTP/1.1" 200 15220
